library/firewall_lib.py

```diff
@@ -251,6 +256,9 @@ def parse_forward_port(module, item):
     return (_port, _protocol, _to_port, _to_addr)
 
 
+def set_the_default_zone(fw, set_default_zone):
```

Comment:
I don't think we need a function for this - just use `fw.setDefaultZone(set_default_zone)` below